src/libs/actions/Report.js

```diff
@@ -1235,27 +1220,18 @@ function subscribeToNewActionEvent(reportID, callback) {
 }
 
 /**
+ * Handles callbacks and notifications for new report actions.
+ *
  * @param {String} reportID
  * @param {Object} action
  */
-function viewNewReportAction(reportID, action) {
+function handleNewReportAction(reportID, action) {
```

Comment:
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#naming-conventions

Reply:
Thanks. Updated to `sendNewReportActionNotification`.